### PR TITLE
Fix #3

### LINKED ISSUE #3: [Fabric API 0.3.0] Registry remapping failed!
When connecting to a server with The Biome Overhaul 1.1.0, you are presented with an error

Registry Remapping Failed

### PR DESCRIPTION
**Pull request by raphydaphy**
You can now join a server which uses the mod. Also changed println to a proper logger.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,17 +1,15 @@
 # Done to increase the memory available to gradle.
 org.gradle.jvmargs=-Xmx1G
 
-# Fabric Properties
-	# check these on https://fabricmc.net/use
-	minecraft_version=1.14
-	yarn_mappings=1.14+build.1
-	loader_version=0.4.2+build.132
+# Fabric Properties (https://modmuss50.me/fabric.html)
```
Comment (valoeghese) [inline gradle.properties]: I see you changed my mappings

Comment (valoeghese) [inline gradle.properties:4]: I see you changed my gradle

Comment (valoeghese) [reply gradle.properties:4]: why did I click review

Comment (valoeghese) [reply gradle.properties:4]: I'm fine with this

Comment (valoeghese) [reply gradle.properties:4]: I just don't know how to use GitHub pull requests properly

Comment (valoeghese) [reply gradle.properties:4]: How do I resolve this

File: src/main/java/valoeghese/biomeoverhaul/ModBiomes.java [hunk 2]
```diff
@@ -39,49 +39,48 @@
...
-	{
+	public static Biome STEPPE;
+	public static Biome SUBTROPICAL_RAINFOREST;
+	public static Biome TEMPERATE_RAINFOREST;
+	public static Biome TUNDRA;
+	public static Biome TROPICAL_ISLAND;
+	public static Biome TROPICAL_ISLAND_SHORE;
+	public static Biome WOODED_MOUNTAIN_PEAKS;
+
+	public static void init() {
```
Comment (valoeghese) [inline src/main/java/valoeghese/biomeoverhaul/ModBiomes.java:83]: I can't accept not having curly braces on new lines

File: src/main/java/valoeghese/biomeoverhaul/world/CustomSurfaceBuilders.java [hunk 3]
```diff
@@ -13,35 +13,35 @@
...
+	public static TernarySurfaceConfig OUTBACK_CONFIG;
 	
 	private static <C extends SurfaceConfig, F extends SurfaceBuilder<C>> SurfaceBuilder<C> register(String string1, F surfaceBuilder1)
 	{
 		return (SurfaceBuilder<C>)Registry.register(Registry.SURFACE_BUILDER, (String)string1, surfaceBuilder1);
 	}
 
-	static
-	{
+	public static void init() {
```
Comment (valoeghese) [inline src/main/java/valoeghese/biomeoverhaul/world/CustomSurfaceBuilders.java:44]: NEW LINE

**Review by valoeghese — CHANGES_REQUESTED** [review]
Curly braces on new lines pls. Yes this is important.

**Comment by valoeghese** [comment]
ok nvm I'll merge this

File: gradle.properties (hunk 1 above)
Comment (raphydaphy) [reply gradle.properties:4]: lol

**Pull request merged**